science/gcal/files/patch-trap-remove-may-month-check-fix.diff

```diff
@@ -0,0 +1,68 @@
+--- src/utils.c.orig	2020-11-27 23:18:52.000000000 +1300
++++ src/utils.c	2020-11-27 23:28:24.000000000 +1300
+@@ -1337,35 +1337,36 @@
+   };
+ 
+ 
+-  if (month == 5)
+-    {
+-      static char *mayname;
+-      static Bool is_initialized = FALSE;
+-
+-
+-      /*
+-         Now let's remove the trailing BLANK character
+-         from the complete text for the month MAY.
+-       */
+-      if (!is_initialized)
+-	{
+-	  register int len;
+-	  auto char *s = _(name[month]);
+-
+-
+-	  len = (int) strlen (s) - 1;
+-	  mayname = (char *) my_malloc (len,
+-					ERR_NO_MEMORY_AVAILABLE,
+-					__FILE__, ((long) __LINE__) - 2L,
+-					"mayname", 0);
+-	  strcpy (mayname, s);
+-	  while (*(mayname + len) == ' ')
+-	    *(mayname + len--) = '\0';
+-	  is_initialized = TRUE;
+-	}
+-
+-      return (mayname);
+-    }
++// MHB 2020-11-27 Causes trap when running Big Sur. Not sure required.
```

Comment:
Don't provide a patch that comments out code. Instead, provide a patch that removes code. Include a reference to the upstream bug report URL about whatever problem is being fixed.

Reply:
Thanks. I have removed the comments and added a return statement.

Now just have to figure out how to the change to show here.